week5/will/taobao/index.css

```diff
@@ -260,30 +259,34 @@ footer {
   padding: 15px 0px;
 }
 
-.footer-firstline a {
+.footer-uperline a {
```

Comment:
spelling is not correct, separate 2 words with hyphens

Reply:
fixed